pages/DSNP/Types/Update.md

```diff
@@ -34,6 +35,11 @@ Updates should be ignored.
 - Resource MUST one of the supported [Activity Content](../../ActivityContent/Overview.md) Types
 - MUST use one of the supported URL Schemes
 
+### targetAnnouncementType
+
+- MUST be the [Announcement Type](../Announcements.md#announcement-types) of the target Announcement
+- MUST ONLY be an Update allowed Announcement Type
+
```

Comment:
We should draw up the `#### Update Allowed Announcement Types` to under this section